include/openssl/opensslconf.h.in

```diff
@@ -95,6 +95,12 @@ extern "C" {
 # define OPENSSL_API_COMPAT OPENSSL_MIN_API
 #endif
 
+#if OPENSSL_API_COMPAT < 0x10200000L
+# define DEPRECATEDIN_1_2_0(f)   DECLARE_DEPRECATED(f)
+#else
+# define DEPRECATEDIN_1_2_0(f)
+#endif
```

Comment:
I seem to be missing something. We already have variables marked as deprecated in 1.2.0, but I can't find how.